src/js/rebuses.js

```diff
@@ -681,6 +681,11 @@ const rebuses = [
     symbols: ['🔥', '+', '🤼', '+', 'er'],
     words: ['firefighter'],
     hint: 'Someone trained to combat fires and rescue people trapped by fires'
+  },
+  {
+    symbols: ['1', '+', 'ce', '+', 'n', '+', 'a', '+', '🔵', '+', '🌛'],
+    words: ['Once in a Blue Moon'],
```

Comment:
If you want your rebus to consist of several words, you can split your words array in to several "one-word" strings 🙂